### PR TITLE
Rename .sz for .count on dtype (and ANETensor for completeness)

### PR DESCRIPTION
**Pull request by fijal**
This renames .sz for .count on dtype, as requested on discord

### CODE REVIEW

File: extra/accel/ane/lib/ane.py [hunk 1]
```diff
@@ -112,16 +112,16 @@ class ANETensor:
   def __init__(self, *shape):
     self.shape = shape
     self.dtype = np.float16
-    self.sz = int(np.prod(shape))
-    assert(self.sz <= 0x4000)
-    self.tt = libane.ANE_TensorCreate(self.sz, 1)
+    self.count = int(np.prod(shape))
```
Comment (geohot) [inline extra/accel/ane/lib/ane.py:115]: WTF

**Comment by geohot** [comment]
Don't change extra

**Comment by fijal** [comment]
Reverted the "ane" changes. Should I also not touch the extra/backends/cstyle.py?

**Comment by Xiloscient** [comment]
none of my business but the making the linter happier by parenthesizing a condition in order to split it into two lines thereby both decreasing readiblity and increasing line count doesn't seem like a good idea

**Comment by fijal** [comment]
I do tend to agree, alternative would be to e.g. use a temp variable, but that would still increase the line count or to disable linter for this line. I'm not familiar with the codebase, so please suggest some solution and I'm happy to do it

**Comment by geohot** [comment]
Extra is still being changed

**Pull request closed without merging**